atak-civ/plugin-examples/plugintemplate/app/src/main/res/layout/preset_timer_cell.xml

```diff
@@ -23,10 +20,7 @@
         android:id="@+id/preset_timer_time"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="10dp"
-        android:layout_marginTop="10dp"
-        android:layout_marginEnd="10dp"
-        android:layout_marginBottom="10dp"
+        android:layout_margin="10dp"
         android:layout_toRightOf="@+id/preset_timer_name"
         android:text="@string/preset_duration_text"
         android:textSize="18sp"
```

Comment:
https://i.imgur.com/mna9tBO.png for some reason buttons aren't aligning when vertical instead of horizontal for me. Not a huge deal though and not sure if this was part of your changes or if itwas like this before. Feel free to ignore this comment if that was the case

Reply:
Thanks for that! I just fixed it.